opl/generators/inventory_egress.py

```diff
@@ -10,11 +10,12 @@
 
 
 class EgressHostsGenerator(opl.generators.generic.GenericGenerator):
-    def __init__(self, count=1, n_packages=300, template='inventory_egress_template.json.j2' , msg_type='created'):
+    def __init__(self, count=1, n_packages=300, template='inventory_egress_template.json.j2' , msg_type='created', per_account_data=[]):
```

Comment:
Hmm, now looking into this, could you please add a docstring above this class or somewhere with some short info about what `perf_account_data` is and example of it (just include account and os tree commit fields in the example - these *_ids fields are not relevant here)? It will help later when we will try to recall what is this about :).